packages/pn-personafisica-webapp/src/components/Contacts/DigitalContact.tsx

```diff
@@ -69,6 +80,8 @@ const DigitalContact = forwardRef<{ toggleEdit: () => void }, Props>(
     const [editMode, setEditMode] = useState(false);
     const contactType = channelType.toLowerCase();
 
+    const EidtButton = slots?.editButton || ButtonNaked;
```

Comment:
Fix this typo here and below
```suggestion
    const EditButton = slots?.editButton || ButtonNaked;
```